src/main/java/hw6/entities/MetalsAndColorsData.java

```diff
@@ -0,0 +1,23 @@
+package hw6.entities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MetalsAndColorsData {
+    private Data data_1;
```

Comment:
This is completely prohibited to read the file in according to amount of data. Just imagine the situation if you need to add one more rows. What will happen in this case ?

Reply:
Yes, it's wrong. Now I have read about TypeToken